src/modules/ProductDetailsPage/ProductDetailsPage.tsx

```diff
@@ -31,26 +29,51 @@ function getDetails(
   if (color && capacity) {
     return productDetail.additional.find(
       (product) => product.color === color && product.capacity === capacity,
-    );
+    ) || null;
   }
```

Comment:
not clear to me why null? We enter in this block when we definatly sure that we have such params. Or we can possibly receive underfind?

Reply:
The find method can return undefined, so to better typify this whole thing, I add null